### ISSUE TITLE
Client change requests for 1.3 release? 

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/natural-selection#257]
From https://github.com/phetsims/natural-selection/issues/251 (1.3 release):

> The client doesn’t need another version in the near-term (before next summer). They will use 1.2 in their beta. If they find any problems, a 1.2 maintenance release may be necessary.

Before we start QA for 1.3... Has the client reported any problems?  Or should someone explicitly ask if they have any change requests?

Assigning to @kathy-phet and @amanda-phet. Work on the 1.3 release needs to start in the next week or two in order to make the delivery milestone, so this is high priority.

**Comment by amanda-phet** [comment]
@kathy-phet emailed the client today to confirm that they don't have any change requests.

**Comment by kathy-phet** [comment]
Here is the reply: 

> Great to hear from you! I was just about to send you a (hopefully) small request, which is to have a label (or option for one?) on the circular pink timer in this sim. Maybe that's not the only/best solution, but I've encountered a couple times with folks that it can be difficult to understand what it does.

>What do you think? Is this something that can be added in relatively easily?

**Comment by kathy-phet** [comment]
Shall we discuss at design meeting?

**Comment by pixelzoom** [comment]
I don't feel like I have enough info to proceed with a design meeting for this request.

@kathy-phet Can you please ask the client for label suggestions, or what they'd like it to communicate?  Does the label need to communicate just "Generation Clock", or does it need to somehow explain how/why the sections of the clock are colored? 

This also seems like the type of request where we'd like to avoid making this an option -- either do it or don't do it.  If we feel the client's concern is legitimate, then presumably it's a legitimate concern in general.

**Comment by kathy-phet** [comment]
I think this has come up some before -- that the generation clock is a bit of a challenge to interpret. Is there a title we could give it that would help generally?

**Comment by amanda-phet** [comment]
I would just label it "Generation". A design meeting would be helpful to determine where to put the label, and if we want it as an option for just PhET-iO or for all users. Once we have the label, the client can always rename it, so I don't think we need to know what they'd want to call it.

**Comment by pixelzoom** [comment]
Intepretation and complexity of the Generation Clock definitely came up before, more than once if I recall correctly.   And we did make a few changes from the original design.

During development, I added the generation number below the clock, because I didn't think it was reasonable to have to look elsewhere to see that the generation number. 

In https://github.com/phetsims/natural-selection/issues/92, we decided to add icons next to the Environmental Factors checkboxes:

<img width="219" alt="screenshot_868" src="https://user-images.githubusercontent.com/3046552/108120519-9e4cc480-705e-11eb-958e-d658ab9461d1.png">

...
 to communicate what the "gray" and "green" slices of the clock signify.  I don't know if this was successful, or whether it was specifically tested in interviews.

Instead of just showing the generation number (eg '2') we could show "Generation 2".   But again, without asking the client a couple more questions, I'm not certain that will address their request.

**Comment by pixelzoom** [comment]
> I would just label it "Generation".

I'm not a fan of a new label. If it's "Generation", then I'd prefer to integrate it with the value, e.g. "Generation 2".

> If it's PhET-iO only, the reality is that it will not receive the same level of testing as a regular feature.

I'm not a fan of a PhET-iO only feature unless that's absolutely necessary.  

> Once we have the label, the client can always rename it, so I don't think we need to know what they'd want to call it.

Why not simply ask the client if "Generation 2" would address the issue for them?  Or ask them what they would suggest - they might have a good idea.

**Comment by pixelzoom** [comment]
https://github.com/phetsims/natural-selection/issues/162#issuecomment-673726376:
> Yes, students understood the generation clock- a full cycle, when mating occurred, and 2/3 easily interpreted the color-coded slices without assistance. While one called a cycle a "day", that could easily be cleared up in a classroom setting, especially since he could tell the x-axis of the population graph was Generations.

So another possibility for addressing this issue would be to share these interview results with the client, and see if that would address their concern.

**Comment by pixelzoom** [comment]
I've moved labeling of the generation clock to its own issue #258, let's please continue the discussion there.  This is it's own feature request, so deserves its own GitHub issue.  And it will make it easier to locate our decisions in the future.

**Comment by pixelzoom** [comment]
It sounds like #258 (Add a label to Generation Clock) was the sole request.  So I'll go ahead and close this issue.
